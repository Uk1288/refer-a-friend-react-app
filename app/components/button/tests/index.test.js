import React from 'react';
import { render } from 'react-testing-library';
import Button from '../index';

describe('<Button />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<Button />);
    expect(firstChild).toMatchSnapshot();
  });
});
