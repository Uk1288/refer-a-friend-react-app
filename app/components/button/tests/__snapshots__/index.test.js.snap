// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render and match the snapshot 1`] = `
<button
  class="button__StyledButton-sc-1bg6su4-0 hygGZF"
/>
`;
